version/version.go

```diff
@@ -1,5 +1,6 @@
 package version
 
+//TODO Check if we can remove it
 var (
 	Version = "7.0.1"
```

Comment:
I think @davidffrench wanted to use it somewhere?

Reply:
ya, i wanted it in the status, the version field is currently blank but I don't mind where it comes from. Also, there will be an enhancement soon to expose the version through the operator metrics endpoint as a new metric. Again, as long as it can be pulled from somewhere.